include/predictors/historic-average-predictor.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
```

Comment:
The file naming is not consistent. Here camelCase and the ``historic-average-predictor.h` with `-`